src/unix/internal.h

```diff
@@ -22,6 +22,12 @@
 #ifndef UV_UNIX_INTERNAL_H_
 #define UV_UNIX_INTERNAL_H_
 
+#if EDOM > 0
+# define UV_ERR(x) (-(x))
+#else
+# define UV_ERR(x) (x)
+#endif
```

Comment:
Can you put this below the includes and `#include <errno.h>` explicitly?

Reply:
[Should be fixed](https://github.com/ohnx/libuv/blob/v1.x-add-uv_err/src/unix/internal.h#L32) now, I think. GitHub doesn't seem to be updating this.